src/components/ProductBacklogListComponent.js

```diff
@@ -330,24 +333,33 @@ const ProductBacklogListComponent = ({ refreshProjects, displayPopup }) => {
     }
   };
 
+  // Only allow access if logged in
+  if (
+    process.env.REACT_APP_USE_AUTH &&
+    !loggedIn &&
+    !sessionStorage.getItem('user')
+  ) {
+    console.log('no user found');
+    return <Redirect to='/login' />;
```

Comment:
I like it, we will have to make sure that it is present on all components that we make going forward